lib/atoma.ts

```diff
@@ -2,8 +2,8 @@ import type { SuiClient } from "@mysten/sui/client";
 import { Transaction } from "@mysten/sui/transactions";
 import type { UseMutateAsyncFunction } from "@tanstack/react-query";
 
-const proxy_url = process.env.NEXT_PUBLIC_PROXY_URL;
-const USDC_TYPE = process.env.NEXT_PUBLIC_USDC_TYPE;
+const proxy_url = "https://credentials.atomacloud.com";
```

Comment:
rever this, and set env values in the `deploy.yml` instead, it's easier to track what's being used if it's outside of the code